src/gui/DecksListView.tsx

```diff
@@ -0,0 +1,170 @@
+import { Platform } from "obsidian";
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import h from "vhtml";
+
+import type SRPlugin from "src/main";
+import { SRSettings } from "src/settings";
+import { COLLAPSE_ICON } from "src/constants";
+import { t } from "src/lang/helpers";
+import { Deck } from "../Deck";
+import {
+    DeckStats,
+    IFlashcardReviewSequencer as IFlashcardReviewSequencer,
+} from "src/FlashcardReviewSequencer";
+import { TopicPath } from "src/TopicPath";
+import { FlashcardModalMode } from "./flashcard-modal";
+
+export class DecksListView {
+    public plugin: SRPlugin;
+    public mode: FlashcardModalMode;
```

Comment:
I don't think this is needed as it is always set to FlashcardModalMode.DecksList

Reply:
I removed this now from the class.